packages/session-replay-browser/src/session-replay.ts

```diff
@@ -74,10 +74,6 @@ export class SessionReplay implements AmplitudeSessionReplay {
     this.joinedConfigGenerator = await createSessionReplayJoinedConfigGenerator(apiKey, options);
     this.config = await this.joinedConfigGenerator.generateJoinedConfig(this.identifiers.sessionId);
 
-    this.loggerProvider.debug(
```

Comment:
This seemed like internal debugging, rather than something that should be customer facing, but lmk if this was meant to be customer facing!

Reply:
I was thinking it might be useful for customers with our now complex joining logic, otherwise it's hard to tell what the end config should look like. Let me know if there's a better way to expose this, though.

Reply:
Got it, I'll just update the message to be a little more user friendly then!

Reply:
Oh that looks much different in the console than I expected, I see its purpose now, ok I'll just prepend the name with session replay